packages/shorebird_cli/lib/src/validators/shorebird_flutter_validator.dart

```diff
@@ -145,26 +145,25 @@ This can cause unexpected behavior if you are switching between the tools and th
         .contains(ShorebirdEnvironment.flutterRevision);
   }
 
-  Future<String> _shorebirdFlutterVersion(ShorebirdProcess process) =>
-      _getFlutterVersion(
-        process: process,
-        checkPathFlutter: false,
-      );
+  Future<String> _shorebirdFlutterVersion(ShorebirdProcess process) {
+    return _getFlutterVersion(process: process);
+  }
 
-  Future<String> _pathFlutterVersion(ShorebirdProcess process) =>
-      _getFlutterVersion(
-        process: process,
-        checkPathFlutter: true,
-      );
+  Future<String> _pathFlutterVersion(ShorebirdProcess process) {
+    return _getFlutterVersion(
+      process: process,
+      useVendedFlutter: false,
+    );
+  }
 
   Future<String> _getFlutterVersion({
     required ShorebirdProcess process,
-    required bool checkPathFlutter,
+    bool useVendedFlutter = true,
```

Comment:
Adjusted this since I was looking at this code and felt it was a bit confusing but happy to revert if folks prefer the old naming.